01-getting-started/installation/platforms/install-on-gcp.md

```diff
@@ -0,0 +1,195 @@
+---
+Description: >-
+  Learn how to quickly deploy Squidex (with MongoDB) on Google Cloud Platform using Cloud Run & MongoDB Atlas.
+---
+
+# Deploy Squidex on Google Cloud Platform using Cloud Run, Cloud Storage & MongoDB Atlas
```

Comment:
Cloud Run is great and I am considering migration as well, but If you have more than once instance you need a dedicated worker, which should be basically another cloud run deployment.

Read more about this: https://squidex.io/post/squidex-7.0-release-candidate-3-released

So:

1. One deployment with autoscaling where CLUSTERING__WORKER=true
2. One deployment without autoscaling and only one instance, where CLUSTERING__WORKER=false

Reply:
Yes, but at least we should mention it before people do something dangerous.

Reply:
We are not leveraging auto-scaling at all if that's what you are referring to, infact I have mentioned minimum and maximum number of instances as 1.
Let me add some info there.